### PR TITLE
Latest Updated Commit on 10th march

### PR DESCRIPTION
**Pull request by Harshan-Nayak**
## Issue that this pull request solves

Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [ ] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [ ] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by VarunGitGood** [comment]
Hey @Harshan-Nayak please add a demo video and explain the code changes

**Review by VarunGitGood — COMMENTED** [review]
Please remove the migration files

**Review by VarunGitGood — CHANGES_REQUESTED** [review]
Please remove the migration files

**Comment by Harshan-Nayak** [comment]
https://github.com/FusionIIIT/Fusion/assets/128978924/ed2c9646-93bc-4e60-8422-5ec32586637b

This is code change video

**Comment by Harshan-Nayak** [comment]
Removed Migration files

**Review by VarunGitGood — CHANGES_REQUESTED** [review]
There are still some migration files

File: FusionIIIT/applications/globals/migrations/0008_auto_20240217_2112.py
Comment (VarunGitGood) [inline FusionIIIT/applications/globals/migrations/0008_auto_20240217_2112.py:1]: we do not need these migration files

File: fusionlab_backup_New.sql
Comment (VarunGitGood) [inline fusionlab_backup_New.sql:1]: Was this asked to be pushed by the database team? @Kushagra-Yadav

**Comment by Harshan-Nayak** [comment]
Also Removed Migration files from globals

**Comment by VarunGitGood** [comment]
@priyansh61 do we not need global migrations as well?

**Comment by VarunGitGood** [comment]
@Harshan-Nayak everything else looks good let me just confirm this.

**Pull request merged**